custom_components/dualmode_generic/climate.py

```diff
@@ -173,7 +185,7 @@ def __init__(
         self._hvac_mode = initial_hvac_mode
         self._saved_target_temp = target_temp or away_temp
         self._temp_precision = precision
-        self._hvac_list = [HVAC_MODE_COOL, HVAC_MODE_HEAT, HVAC_MODE_OFF]
+        self._hvac_list = [HVAC_MODE_COOL, HVAC_MODE_HEAT, HVAC_MODE_FAN_ONLY, HVAC_MODE_OFF]
```

Comment:
Given the ventilator entity is optional, the `_hvac_list` field needs to exclude the `HVAC_MODE_FAN_ONLY` if a ventilator value isn't provided.

Reply:
Thanks for your feedback! How can I implement a check for whether or not the ventilator is loaded?

Reply:
Ok, I added all the necessary checks of ventilator_entity_id to get the actually have the ventilator be optional

_Took me a lot longer than I hoped it would 🤯_